doc/ref_model/chapters/chapter03.md

```diff
@@ -508,12 +509,74 @@ Figure 3-10 shows a simple architecture of an SFC with multiple VNFs, as SF data
 <p align ="center"><b>Figure 3-10:</b> SFC Architecture for VNF based SFs </p>
 
 
-Figure 3-11 shows a simple architecture of an SFC with multiple CNFs, as SF data plane components, along with SFC management and CNF MANO components. 
+Figure 3-11a shows a simple architecture of an SFC with multiple CNFs, as SF data plane components, along with SFC management and CNF MANO components. 
 <p align="center"> <img src="../figures/ch03-model-sfc-architecture-cnf-2.png" alt="SFC Architecture for CNF based SFs" Title="SFC Architecture for CNF based SFs" width="45%"/></p>
-<p align ="center"><b>Figure 3-11:</b> SFC Architecture for CNF based SFs</p>
+<p align ="center"><b>Figure 3-11a:</b> SFC Architecture for CNF based SFs</p>
```

Comment:
@CsatariGergely i made it a b to remove collision as suggested . I think i can take up once we dont have any comments before final merge to align the figure numbering. 
For figure, i used draw.io and i have xml file to share .. but i can not have link in document and it will be lost in folder. Please let me know if you want file , i can attach in the comments section.